src/objects/zcl_abapgit_object_shi5.clas.abap

```diff
@@ -144,8 +155,14 @@ CLASS ZCL_ABAPGIT_OBJECT_SHI5 IMPLEMENTATION.
              INTO TABLE ls_extension-texts
              WHERE extension = mv_extension.
 
+    SELECT * FROM ttrees
+            INTO TABLE ls_extension-sequences
+            WHERE extension = mv_extension.
```

Comment:
please add ORDER BY PRIMARY KEY, to secure sequence on HANA systems

Reply:
Added to selection for both tables